Content.Client/Entry/IgnoredComponents.cs

```diff
@@ -323,7 +323,13 @@ public static class IgnoredComponents
             "GrowingKudzu",
             "MonkeyAccent",
             "ReplacementAccent",
-            "ResistLocker"
+            "ResistLocker",
+            "SpawnArtifact",
+            "TelepathicArtifact",
+            "ArtifactGasTrigger",
+            "ArtifactInteractionTrigger",
+            "Artifact",
+            "RandomArtifactSprite"
```

Comment:
don't need to change anything here but in the future you'll usually get less conflicts if you choose a random place in the list to put these rather than just the end

Reply:
Ok, will do next time